shopfloor_reception_packaging_dimension/services/reception.py

```diff
@@ -28,14 +28,30 @@ def _before_state__set_quantity(self, picking, line, message=None):
     def _get_domain_packaging_needs_dimension(self):
         return expression.OR(
             [
-                [("packaging_length", "=", 0)],
-                [("packaging_length", "=", False)],
-                [("width", "=", 0)],
-                [("width", "=", False)],
-                [("height", "=", 0)],
-                [("height", "=", False)],
-                [("weight", "=", 0)],
-                [("weight", "=", False)],
+                [
+                    "|",
+                    ("packaging_length", "=", 0),
+                    ("packaging_length", "=", False),
+                    ("packaging_level_id.shopfloor_collect_length", "=", True),
```

Comment:
looks like `packaging_level_id.shopfloor_collect_length` should stay outside in an AND? :thinking: